### ISSUE TITLE
Average Precision and Average Recall on DETR using Visdrone2019 are all 0

### ISSUE THREAD

**Issue by Seraths** [open-mmlab/mmdetection#5507]
I encountered this when I was training on colab, using `detr` and visdrone2019 dataset, this is the evaluation after 2 epochs.

Is there any possible reasons that this may occur? I trained the dataset using cascade rcnn and it was all normal, not sure why this is happening to `detr`.

![image](https://user-images.githubusercontent.com/26945061/124220114-b2dab680-db40-11eb-8f99-2a07b6dc848c.png)

**Comment by Seraths** [comment]
The one I said results looks normal was a month ago. Today, on my setup (colab), I have lr = 0.02, I tried on other rcnns and cascade rcnns they all return AP and AR around 0.001~0.005 for the first epoch. Is this normal?

**Comment by tikitong** [comment]
Hi @Seraths may be similar case here #5500, **detr** performance is extremely low on all datasets I tested, can you share your config file?

**Comment by Seraths** [comment]
> Hi @Seraths may be similar case here #5500, **detr** performance is extremely low on all datasets I tested, can you share your config file?

This is what I have in colab cell:
```
cfg.optimizer.lr = 0.02 / 8

cfg.evaluation.metric = 'bbox'
cfg.evaluation.interval = 1

cfg.seed = 0
set_random_seed(0, deterministic=False)
cfg.gpu_ids = range(1)
```

Inside `detr_r50_8x2_150e_coco.py`, I only changed one line, which is inside `model` 's `bbox_head`:
`num_classes=10,`

I haven't test it with full epoch size, but the first epoch today is still giving me all 0.

**Comment by tikitong** [comment]
> > Hi @Seraths may be similar case here #5500, **detr** performance is extremely low on all datasets I tested, can you share your config file?
> 
> This is what I have in colab cell:
> 
> ```
> cfg.optimizer.lr = 0.02 / 8
> 
> cfg.evaluation.metric = 'bbox'
> cfg.evaluation.interval = 1
> 
> cfg.seed = 0
> set_random_seed(0, deterministic=False)
> cfg.gpu_ids = range(1)
> ```
> 
> Inside `detr_r50_8x2_150e_coco.py`, I only changed one line, which is inside `model` 's `bbox_head`:
> `num_classes=10,`
> 
> I haven't test it with full epoch size yet.

It seems that converging with `DETR` requires a large learning rate on the first 10 epochs, maybe test with `cfg.optimizer.lr = 0.02 `even if you only use 1 GPU.

**Comment by Seraths** [comment]
> > > Hi @Seraths may be similar case here #5500, **detr** performance is extremely low on all datasets I tested, can you share your config file?
> > 
> > 
> > This is what I have in colab cell:
> > ```
> > cfg.optimizer.lr = 0.02 / 8
> > 
> > cfg.evaluation.metric = 'bbox'
> > cfg.evaluation.interval = 1
> > 
> > cfg.seed = 0
> > set_random_seed(0, deterministic=False)
> > cfg.gpu_ids = range(1)
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Inside `detr_r50_8x2_150e_coco.py`, I only changed one line, which is inside `model` 's `bbox_head`:
> > `num_classes=10,`
> > I haven't test it with full epoch size yet.
> 
> It seems that converging with `DETR` requires a large learning rate on the first 10 epochs, maybe test with `cfg.optimizer.lr = 0.02 `even if you only use 1 GPU.

`DETR` immediately went to `NaN` during the first epoch after I change the learning rate to 0.02.

Thanks for the advice though, I also tried on other more matured models and results are quite low as well. I shall reconsider if this is an issue with my dataset.

**Comment by jshilong** [comment]
Feel free to reopen the issue if there is any question
